sophia-vote-contract.md

```diff
@@ -0,0 +1,142 @@
+IAL: How to create a Sophia fungible token contract?
```

Comment:
whats IAL?

Reply:
Hahaha (classic) DOUBLE copy paste fail - took the TUTORIAL: How to create a Sophia fungible token contract? as inspiration and just realized now that I totally missed to adjust the headline as well. FAIL! lol *facepalm*